DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/DiagnosticContentViewModelTests.cs

```diff
@@ -161,6 +163,42 @@ public void Diagnostic_content_should_parse_html_url()
             );
         }
 
+        [Test]
+        public void Diagnostic_content_should_parse_html_url_using_all_tutorials_when_can_select_tutorials_is_false()
+        {
+            // Given
+            const int currentVersion = 55;
+            const int postLearningPassThreshold = 77;
+            const string diagnosticAssessmentPath = "https://www.dls.nhs.uk/CMS/CMSContent/Course119/Diagnostic/07DiagnosticTesting/itspplayer.html";
+            var selectedTutorials = new List<int>(new[] { 1, 2, 3 });
```

Comment:
These should all be tutorials which are part of the content for a more realistic test 🙂